src/GeneralStateTestsFiller/stEWASMTests/utils/revertFiller.yml

```diff
@@ -0,0 +1,159 @@
+revert:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    # tx sender
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: ''
+      storage: {}
+    # called contract
+    deadbeef00000000000000000000000000000000:
+      balance: '100000000000'
+      code: |
+        (module
+          (import "ethereum" "revert" (func $revert (param i32 i32)))
+          (memory 1)
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+          (func $main
+            ;; Store some data to return
+            (i32.store (i32.const 0) (i32.const 0xabcd))
+            ;; Revert and return it
+            (call $revert
```

Comment:
This doesn't seem like correct behavior.  `revert` should act like an EVM exception but without consuming all of the gas provided for the transaction.  Everything after this statement should not be executed...

Reply:
Also add some state changing operation afterwards, and then add another expect condition that ensures that the second state changing operation was not executed.

Reply:
Done